litmus-portal/backend/graphql-server/pkg/database/mongodb/operations/project.go

```diff
@@ -0,0 +1,62 @@
+package operations
+
+import (
+	"context"
+	"log"
+
+	"github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/pkg/database/mongodb"
+	dbSchema "github.com/litmuschaos/litmus/litmus-portal/backend/graphql-server/pkg/database/mongodb/schema"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+var projectCollection *mongo.Collection
+
+func init() {
```

Comment:
Init all the collections in one place in the `initAllCollections` functions, also declare all the collections in the init file.

Reply:
In this approach it wouldn't be required to have all the collections in the init file. We will just have to create a separate file in order to add any resource